src/common/components/Badge/index.tsx

```diff
@@ -0,0 +1,71 @@
+import { VariantProps } from 'class-variance-authority';
+import { ComponentProps } from 'react';
+
+import { cn } from '~/utils/cn';
+
+import {
+  badgePositionVariants,
+  badgeSizeVariants,
+  badgeVariants,
+} from './Badge.variants';
+
+interface BadgeProps
+  extends VariantProps<typeof badgeVariants>,
+    VariantProps<typeof badgeSizeVariants>,
+    VariantProps<typeof badgePositionVariants>,
+    ComponentProps<'img'> {
+  children: React.ReactNode;
+  badgeType: 'online' | 'offline' | 'alarm';
+  isActive?: boolean;
+}
+
+const Badge = ({
+  children,
+  badgeType,
+  isActive,
+  corner,
+  size,
+  className,
+  translateX,
+  translateY,
+  ...props
+}: BadgeProps) => {
+  const getBadgeStyles = () => {
+    switch (badgeType) {
+      case 'online':
+        return 'bg-success';
+      case 'offline':
+        return ' bg-gray-700 animate-none';
```

Comment:
```suggestion
        return 'bg-gray-700 animate-none';
```


`offline`일 때 `animate-none`은 `animate-ping`을 없애는 걸까요? 맞다면 적용이 안 되고 있는 것 같아요!